asreview/state/sqlstate.py

```diff
@@ -620,24 +620,19 @@ def add_labeling_data(self, record_ids, labels, notes=None, prior=False):
         con.commit()
         con.close()
 
-    def change_decision(self, record_id):
+    def update_decision(self, record_id, label, note=None):
         """Change the label of a record from 0 to 1 or vice versa."""
-        current_label = self.get_data_by_record_id(record_id)['label'][0]
-
-        # New label should be of type int, not np.int, to insert into sql.
-        new_label = int(1 - current_label)
-        current_time = datetime.now()
 
         con = self._connect_to_sql()
         cur = con.cursor()
 
         # Change the label.
-        cur.execute("UPDATE results SET label = ? WHERE record_id = ?",
-                    (new_label, record_id))
+        cur.execute("UPDATE results SET label = ?, notes = ? WHERE record_id = ?",
+                    (label, note, record_id))
 
         # Add the change to the decision changes table.
         cur.execute("INSERT INTO decision_changes VALUES (?,?, ?)",
-                    (record_id, new_label, current_time))
+                    (record_id, label, datetime.now()))
```

Comment:
Do we want to update this table if we only add a note, but keep the label the same? Or we could add the notes to the decision_changes table, that way you also save the history of the notes. Note sure if that is desirable though.

Reply:
I was thinking about privacy-related issues. A decision doesn't harm anyone. However, a note can contain info like "Discuss with John Doe". The note is removed by the reviewer after the discussion, however, it's never removed from the history.

Reply:
Yes, that makes sense. I agree that you should be able to remove notes, but not the previous labels.